### PR TITLE
Fix #107: Add remove button (styled as an 'X') to ca-object-list

### LINKED ISSUE #107: ca-object-list needs to allow objects to be removed as well as added
Add the moment it just has an 'add' button. It should probably behave more like ca-input-array.

### PR DESCRIPTION
**Pull request by tapina**

### CODE REVIEW

File: components/ca-object-list.html [hunk 1]
```diff
@@ -9,11 +9,17 @@
     ca-object-list {
         display: block;
         position: relative;
-        overflow: hidden;
+        overflow: inherit;
```
Comment (kevmarchant) [inline components/ca-object-list.html]: shouldn't need overflow: inherit - it should be implicit, just removing the hidden should work fine

Comment (tapina) [reply components/ca-object-list.html]: I'll try it

Comment (tapina) [reply components/ca-object-list.html]: Removed

**Comment by tapina** [comment]
TYE-614

**Pull request merged**